packages/yew-macro/tests/html_macro/component-fail.stderr

```diff
@@ -386,6 +386,20 @@ error: only an expression may be assigned as a property. Consider removing this
 148 |     html! { <HtmlInProps header={format!("ending with semi");} /> };
     |                                                             ^
 
+error: mismatched closing tags: expected `A`, found `B`
+   --> tests/html_macro/component-fail.rs:177:22
+    |
+177 |     let _ = html! { <A></B> };
+    |                      ^^^^^
+
+error: expected a valid closing tag (e.g.: </Component>)
+   --> tests/html_macro/component-fail.rs:178:13
+    |
+178 |     let _ = html! { <A></> };
+    |             ^^^^^^^^^^^^^^^^
```

Comment:
This error should have a span associated with it, probably `</>`

Reply:
Should be good now